### PR TITLE
Add sudo calls to migrate data in spaces and space-follows pallets

### PR DESCRIPTION
**Pull request by F3Joule**

### CODE REVIEW

**Review by siman — APPROVED** [review]

File: pallets/spaces/src/lib.rs [hunk 1]
```diff
@@ -308,6 +308,54 @@ pub mod pallet {
...
+            };
+
+            SpaceById::<T>::insert(space_id, new_space);
+            SpaceIdsByOwner::<T>::mutate(&owner, |ids| {
+                ids.try_push(space_id).expect("qed; too many spaces per account")
+            });
+
+            Self::deposit_event(Event::SpaceCreated(owner, space_id));
+
+            Ok(())
```
Comment (TarekkMA) [inline pallets/spaces/src/lib.rs:350]: Add Pays::No

File: pallets/spaces/src/lib.rs [hunk 2]
```diff
@@ -308,6 +308,54 @@ pub mod pallet {
...
+        }
+
+        #[pallet::weight(10_000 + T::DbWeight::get().writes(1))]
+        pub fn force_set_next_space_id(origin: OriginFor<T>, space_id: SpaceId) -> DispatchResult {
+            ensure_root(origin)?;
+            NextSpaceId::<T>::put(space_id);
+            Ok(())
```
Comment (TarekkMA) [inline pallets/spaces/src/lib.rs:357]: Add Pays::No

**Review by TarekkMA — CHANGES_REQUESTED** [review]

File: pallets/space-follows/src/lib.rs [hunk 3]
```diff
@@ -134,6 +134,29 @@ pub mod pallet {
...
+            space_id: SpaceId,
+        ) -> DispatchResultWithPostInfo {
+            ensure_root(origin)?;
+
+            SpaceFollowers::<T>::mutate(space_id, |followers| followers.push(follower.clone()));
+            SpaceFollowedByAccount::<T>::insert((follower.clone(), space_id), true);
+            SpacesFollowedByAccount::<T>::mutate(follower.clone(), |space_ids| {
+                space_ids.push(space_id)
+            });
+
```
Comment (TarekkMA) [inline pallets/space-follows/src/lib.rs:155]: Can we use `HashSet` instead of `Vec`, so we can avoid duplicates.

File: pallets/spaces/src/lib.rs [hunk 4]
```diff
@@ -308,6 +308,62 @@ pub mod pallet {
...
+                owner: owner.clone(),
+                parent_id: None,
+                content,
+                hidden,
+                posts_count: 0,
+                permissions,
+            };
+
+            SpaceById::<T>::insert(space_id, new_space);
+            SpaceIdsByOwner::<T>::mutate(&owner, |ids| {
```
Comment (TarekkMA) [inline pallets/spaces/src/lib.rs:348]: See if we can use `HashSet`

**Review by TarekkMA — APPROVED** [review]

**Review by siman — APPROVED** [review]

**Pull request merged**